src/main/resources/application.yml

```diff
@@ -6,6 +6,13 @@ application:
     displayResponseContent: false
   incomingRequests:
     allowedMethodTypes: [ POST, GET ]
+  httpCodesAllowList: [ 200, 201, 202 ]
+  defaultAction:
+    service: default-action
+    body:
+      some_val: "Hello World"
+    query:
+      another_val: 123
```

Comment:
I updated the Acceptance Criteria of https://github.com/buerokratt/Ruuter/issues/26 by adding _"Request ID of a failed response is passed on to Default Action Endpoint so that Logs Analyst could locate appropriate logs for further processing."_